src/templates/base/2019/base_ebook.html

```diff
@@ -27,125 +32,127 @@
 {% endblock %}
 
 {% block content %}
-<section class="content">
+<div class="content">
 
   <section id="title">
-      <h1 class="title title-center">{{ self.intro_title() }}</h1>
-      <h2 class="title title-center">{{ self.intro_sub_title() }}</h2>
-
-      <img class="intro-image" src="/static/images/home-hero.png" alt="" width="984" height="354" loading="lazy" />
-    </section>
-
-    <section>
-      <h1>{{ self.about_title() }}</h1>
-
-      {{ self.mission() }}
-
-      <div class="contributors">
-        <h2 class="title">{{ self.contributors_title() }}</h2>
-        <p>{{ self.contributors_description() }}
-          <a href="#contributors" class="alt">
-            {{ self.contributors_link() }}
-          </a>
-        </p>
-      </div>
-
-      <div class="methodology">
-        <h2 class="title">{{ self.methodology_title() }}</h2>
-        <p class="methodology-info">
-          {{ self.methodology_description() }}
-          <a href="#methodology" class="alt">
-            {{ self.methodology_link() }}
-          </a>
-        </p>
-        <img class="methodology-characters" src="/static/images/methodology-characters.png" alt="" width="984" height="354" loading="lazy" />
+    <h1 class="title title-center">{{ self.intro_title() }}</h1>
+    <h2 class="title title-center">{{ self.intro_sub_title() }}</h2>
+
+    <img class="intro-image" src="/static/images/home-hero.png" alt="" width="984" height="354" />
```

Comment:
This image should have a transparent background but appears with a gray background in the PDF. Not sure what we can do about that.
![image](https://user-images.githubusercontent.com/1120896/82123795-44cf0680-9769-11ea-9d17-48042fa93bd8.png)

Reply:
This was a deliberate choice and I added the grey with CSS:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/fe540e7f0f35a932d42a9c362bcc75e19bcb6053/src/static/css/ebook.css#L45-L50

Without this, the image blends into the background too much:

![image](https://user-images.githubusercontent.com/10931297/82124303-187daf00-9796-11ea-82f4-ab06aa82f90c.png)

Reply:
Ah ok. The transparency may not be an issue if we go with https://github.com/HTTPArchive/almanac.httparchive.org/pull/833#discussion_r426164222

Reply:
Resolving this as discussing below.